src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
+    public static String getRandomColor() {
+        int randNumber = new Random().nextInt(Color.values().length);
```

Comment:
Instantiating a new `Random` object every time the method is called is inefficient. Consider creating a single `Random` instance as a class-level variable and reusing it.

Reply:
fix this case about random